pages/add/form/detail-info-fields.js

```diff
@@ -80,15 +51,17 @@ let Tags = React.createClass({
     }).filter(suggestion => !!suggestion);
   },
   render: function() {
+    let delimiters = [9,13,32,188]; // keycodes for tab,enter,space,comma
```

Comment:
Yeah okay this is weird. Let's take out "32" for space because spaces in tags should be fine (e.g. "digital literacy"), and move it up as a `const DELIMITERD = [...]` and then it's R+